fix(PHP): Clarify browser auto-instrumentation (#15436)
newrelic/docs-website: src/content/docs/apm/agents/php-agent/features/browser-monitoring-php-agent.mdx

freshnessValidatedDate: never
---

> With the PHP agent, you can add [browser monitoring](/docs/browser/new-relic-browser/getting-started/introduction-new-relic-browser) to your pages either automatically or manually. By default, the PHP agent automatically injects the browser agent JavaScript snippet into your webpages. This means you'll get front-end browser performance out of the box without needing to manually insert the JavaScript snippet. You can enable or disable auto-instrumentation in your config file, or you can use the PHP agent API to manually configure browser monitoring.

Reviewer: ```suggestion
By default, the PHP agent automatically adds [browser monitoring](/docs/browser/new-relic-browser/getting-started/introduction-new-relic-browser) to your application's web pages. In this automated process, the PHP agent injects the browser agent JavaScript snippet into your webpages. This means you get front-end browser performance monitoring without needing to manually insert the JavaScript snippet into your page's head element. 

You can enable or disable auto-instrumentation in your PHP agent config file, or you can use the PHP agent API to manually configure browser monitoring.
```

Tried to update this to clarify what's happening here.

## Use or disable auto-instrumentation [#auto_instrumentation]

> To configure the agent to automatically instrument pages with the browser monitoring script, add or edit the [`newrelic.browser_monitoring.auto_instrument`](/docs/apm/agents/php-agent/configuration/php-agent-configuration/#inivar-autorum) setting in your INI file. The PHP agent will automatically inject the browser monitoring's JS snippet into your pages.

Reviewer: ```suggestion
By default, the PHP agent automatically injects the browser monitoring's JavaScript snippet into your pages. To change this setting,  edit the [`newrelic.browser_monitoring.auto_instrument`](/docs/apm/agents/php-agent/configuration/php-agent-configuration/#inivar-autorum) setting in your INI file. 
```
Thought this made more sense, given the header change.
